src/components/vm/snapshots/vmSnapshotsCreateModal.jsx

```diff
@@ -30,9 +30,11 @@ import { DialogsContext } from 'dialogs.jsx';
 import { ModalError } from "cockpit-components-inline-notification.jsx";
 import { FileAutoComplete } from "cockpit-components-file-autocomplete.jsx";
 import { snapshotCreate, snapshotGetAll } from "../../../libvirtApi/snapshot.js";
-import { getSortedBootOrderDevices, LIBVIRT_SYSTEM_CONNECTION, dirname } from "../../../helpers.js";
+import { getSortedBootOrderDevices, LIBVIRT_SYSTEM_CONNECTION } from "../../../helpers.js";
 import { domainGet } from '../../../libvirtApi/domain.js';
 
+import { dirname } from "cockpit-path";
```

Comment:
Wrong sorting.. this should move up to the other cockpit lib imports, and then there should be a newline before the c-machines local imports.

Reply:
I can fix that up, I got too used to our files eslint automatically fixing the import grouping